test/client_worker_test.cc

```diff
@@ -34,17 +34,23 @@ class ClientWorkerTest : public Test {
     benchmark_client_ = new MockBenchmarkClient();
     sequencer_ = new MockSequencer();
     header_generator_ = new MockHeaderSource();
+    termination_predicate_ = new MockTerminationPredicate();
+
     EXPECT_CALL(benchmark_client_factory_, create(_, _, _, _, _, _, _))
         .Times(1)
         .WillOnce(Return(ByMove(std::unique_ptr<BenchmarkClient>(benchmark_client_))));
 
-    EXPECT_CALL(sequencer_factory_, create(_, _, _, _))
+    EXPECT_CALL(sequencer_factory_, create(_, _, _, _, _, _))
         .Times(1)
         .WillOnce(Return(ByMove(std::unique_ptr<Sequencer>(sequencer_))));
 
     EXPECT_CALL(header_generator_factory_, create())
         .Times(1)
         .WillOnce(Return(ByMove(std::unique_ptr<HeaderSource>(header_generator_))));
+
+    EXPECT_CALL(termination_predicate_factory_, create(_, _, _))
+        .Times(1)
```

Comment:
Nit: `Times(1)` is redundant, it's implied by just the basic `EXPECT`.

Reply:
Removed